extensions/sql-database-projects/README.md

```diff
@@ -1,39 +1,57 @@
+SQL Database Projects for Azure Data Studio and VS Code provides a way to design, edit, and publish objects to SQL databases from a source controlled project. For a complete development workflow, build and deploy your database projects in CI/CD pipelines, such as [GitHub Actions](https://github.com/azure/sql-action) or Azure DevOps.
 
-Learn more about the SQL Database Projects extension in the documentation: https://aka.ms/azuredatastudio-sqlprojects
+![SQL project lifecycle](https://raw.githubusercontent.com/microsoft/azuredatastudio/main/extensions/sql-database-projects/images/sqlproj-lifecycle.png)
 
-## Features
+📕 Learn more about SQL Database Projects in the documentation: https://aka.ms/sqlprojects
 
-- Develop database objects using T-SQL
-- Store the database schema in source control
-- Validate object relationships with project build
-- Publish the database objects to a SQL Server or Azure SQL instance
-- Publish the database objects to a local development container
-- Update the database project from a database
+## Requirements
 
-### Preview Features
+The extension will prompt to install the [.NET SDK](https://aka.ms/sqlprojects-dotnet) if it is not found, as it is required to build SQL projects. You can also install the .NET SDK manually from [here](https://aka.ms/sqlprojects-dotnet) for Windows, macOS, and Linux.
 
-- Microsoft.Build.Sql SDK-style projects
-- Generate SQL projects from OpenAPI/Swagger specs
+## Quick start
+
+1. Create a new database project by going to the Database Projects view or by searching for Database Projects: New in the command palette.
+2. Add `.sql` files to the project to define database objects. For example, creating a file `Product.sql` in the SQL project folder to add a table named `Product`:
+  ```sql
+  CREATE TABLE [dbo].[Product](
+      [ProductID] [int] IDENTITY(1,1) NOT NULL PRIMARY KEY,
+      [ProductName] [nvarchar](200) NOT NULL
+  );
+  ```
+3. Build the SQL project to validate the SQL syntax and object references. In the Database Projects view, right-click the database project's root node and select Build.
+4. Building the project created a `.dacpac` file in the output folder. This file contains the schema of the database project and can be deployed to a SQL Server or Azure SQL instance.
 
+📕 Dive in further with this complete tutorial for [creating and deploying a SQL project](https://learn.microsoft.com/sql/tools/sql-database-projects/tutorials/create-deploy-sql-project?pivots=sq1-visual-studio-code).
 
-![AdventureWorks SQL project](https://raw.githubusercontent.com/microsoft/azuredatastudio/main/extensions/sql-database-projects/images/readme-sqlproj.png)
```

Comment:
Would you want to delete this .png as well since it's no longer being referenced?

Reply:
switched it to use an image of the same name, essentially swapping in place